constructs/elasticsearch.ts

```diff
@@ -56,16 +56,6 @@ curl 127.0.0.1:9200/_template/zero-replicas -s --fail -XPUT -H 'Content-Type: ap
         },
         antiAffinity: 'soft',
         esJavaOpts: '-Xmx128m -Xms128m -Dlog4j2.formatMsgNoLookups=true',
```

Comment:
You also need to change this environment variables

Reply:
Thanks!